src/playout/mod.rs

```diff
@@ -75,14 +75,11 @@ pub trait Playout: Sync + Send {
             loop {
                 let first = playable_move.unwrap();
```

Comment:
Ah yes, I see. Some day I'll remove all that again to see if it even makes a performance difference. I have an aversion to complicated code, but if it's faster then I'm all for it.